lib/elixir/lib/kernel.ex

```diff
@@ -525,15 +526,16 @@ defmodule Kernel do
       :b
 
   """
-  @spec max(first, second) :: (first | second) when first: term, second: term
-  def max(first, second) do
-    :erlang.max(first, second)
+  @spec max(left, right) :: (left | right) when left: term, right: term
+  def max(left, right) do
+    :erlang.max(left, right)
```

Comment:
I think I like `value1` slightly more than `first`, maybe because it visually keeps variables close. :man_shrugging: